Dockerfile

```diff
@@ -93,8 +93,8 @@ RUN set -ex && cd ~ \
   && rm -r awscliv2.zip awscliv2.sig aws
 
 #install goreleaser
-ARG GORELEASER_VERSION=0.128.0
-ARG GORELEASER_SHA256SUM=2d9bcff7612700a2a9fe4a085a7f1a84298c2f4d70eab50b1eb5aa5d7863f7c4
+ARG GORELEASER_VERSION=0.134.0
+ARG GORELEASER_SHA256SUM=b4c672caaeae7761104680ba8a9a20d9b08ed912cdaf095ceea3312bc9880815
```

Comment:
How does goreleaser connect to dependabot, or are they totally separate?

Does goreleaser make it so I don't need to remove the executables from the bin?

Reply:
> How does goreleaser connect to dependabot, or are they totally separate?
> 
They are distinct tools. 
> Does goreleaser make it so I don't need to remove the executables from the bin?
I'm not sure what you're what your asking here.

Reply:
A better question might be: is there some overlap with import handling between goreleaser and dependabot? Or does goreleaser just init the go.mod/.sum files?

I mostly don't understand how we're managing dependencies, I guess.